!!python/object:huggingface_hub.community.DiscussionWithDetails
author: ryusangwon
conflicting_files: null
created_at: 2023-06-21 08:36:58+00:00
diff: null
endpoint: https://huggingface.co
events:
- !!python/object:huggingface_hub.community.DiscussionComment
  _event:
    author:
      avatarUrl: https://aeiljuispo.cloudimg.io/v7/https://cdn-uploads.huggingface.co/production/uploads/6313fe1de29fb2e86d62dacc/u0EFbO8mqNvoev8MPeOac.jpeg?w=200&h=200&f=face
      fullname: Ryu Sangwon
      isHf: false
      isOrgMember: false
      isOwner: false
      isPro: false
      name: ryusangwon
      type: user
    createdAt: '2023-06-21T09:36:58.000Z'
    data:
      edited: false
      editors:
      - ryusangwon
      hidden: false
      identifiedLanguage:
        language: en
        probability: 0.8064209222793579
      isReport: false
      latest:
        author:
          avatarUrl: https://aeiljuispo.cloudimg.io/v7/https://cdn-uploads.huggingface.co/production/uploads/6313fe1de29fb2e86d62dacc/u0EFbO8mqNvoev8MPeOac.jpeg?w=200&h=200&f=face
          fullname: Ryu Sangwon
          isHf: false
          isPro: false
          name: ryusangwon
          type: user
        html: "<p>Hi,</p>\n<p>I'm trying to evaluate language model dataset with llama</p>\n\
          <p>but I can't load the model, due to this error</p>\n<p><a rel=\"nofollow\"\
          \ href=\"https://cdn-uploads.huggingface.co/production/uploads/6313fe1de29fb2e86d62dacc/r5g3yBznJ2B9EKT8ys8Kh.png\"\
          ><img alt=\"\u1109\u1173\u110F\u1173\u1105\u1175\u11AB\u1109\u1163\u11BA\
          \ 2023-06-21 \u110B\u1169\u1112\u116E 6.36.35.png\" src=\"https://cdn-uploads.huggingface.co/production/uploads/6313fe1de29fb2e86d62dacc/r5g3yBznJ2B9EKT8ys8Kh.png\"\
          ></a></p>\n<p>how can I deal with it?</p>\n"
        raw: "Hi,\r\n\r\nI'm trying to evaluate language model dataset with llama\r\
          \n\r\nbut I can't load the model, due to this error\r\n\r\n![\u1109\u1173\
          \u110F\u1173\u1105\u1175\u11AB\u1109\u1163\u11BA 2023-06-21 \u110B\u1169\
          \u1112\u116E 6.36.35.png](https://cdn-uploads.huggingface.co/production/uploads/6313fe1de29fb2e86d62dacc/r5g3yBznJ2B9EKT8ys8Kh.png)\r\
          \n\r\nhow can I deal with it?"
        updatedAt: '2023-06-21T09:36:58.524Z'
      numEdits: 0
      reactions: []
    id: 6492c4bac416da73cd1fc0a8
    type: comment
  author: ryusangwon
  content: "Hi,\r\n\r\nI'm trying to evaluate language model dataset with llama\r\n\
    \r\nbut I can't load the model, due to this error\r\n\r\n![\u1109\u1173\u110F\u1173\
    \u1105\u1175\u11AB\u1109\u1163\u11BA 2023-06-21 \u110B\u1169\u1112\u116E 6.36.35.png](https://cdn-uploads.huggingface.co/production/uploads/6313fe1de29fb2e86d62dacc/r5g3yBznJ2B9EKT8ys8Kh.png)\r\
    \n\r\nhow can I deal with it?"
  created_at: 2023-06-21 08:36:58+00:00
  edited: false
  hidden: false
  id: 6492c4bac416da73cd1fc0a8
  type: comment
is_pull_request: false
merge_commit_oid: null
num: 3
repo_id: huggyllama/llama-7b
repo_type: model
status: open
target_branch: null
title: 'how to deal with KeyError: ''llama''?'
